pkg/controller/configmap/configmap_controller.go

```diff
@@ -210,6 +217,15 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 
 		if !ncpImageChanged {
 			log.Info("no new configuration needs to apply")
+            // Check if network config status must be updated
```

Comment:
is the indentation correct here?

Reply:
I'll verify on the source code.

Reply:
Indentation is correct but my editor is using spaces instead of tabs... hence the problem.
Let me fix it.